.github/workflows/main.yml

```diff
@@ -46,3 +46,19 @@ jobs:
 
       - name: Go test
         run: go test ./...
+      - 
+        name: Prepare artifact for Deployer
+        uses: actions/upload-artifact@v2
+        with:
+          name: polygon-sdk_${{ github.sha }}
```

Comment:
Please include the branch name as well as a single commit may be in multiple branches.

Reply:
Branch name added in commit 009a4c88b62d832613c3fe6bce2562f24c010b34

Reply:
Fixed artefact name error ( "/" invalid char ) in commit b608dce2e9433337c79e3196feb6b8546f223024